Assets/RandomLevelGenerator.cs

```diff
@@ -0,0 +1,68 @@
+using UnityEngine;
+using UnityEngine.Assertions;
+
+public class RandomLevelGenerator : MonoBehaviour
+{
+    public uint pathPiecesCount;
```

Comment:
`uint`s are good in the sense that they do not allow negative numbers. However, they are hard to work with because all APIs work with `int`s.

If you want to limit the number in the Inspector, use `[Min(0)]`